src/bijectors/stacked.jl

```diff
@@ -21,12 +21,13 @@ b = stack(b1, b2)
 b([0.0, 1.0]) == [b1(0.0), 1.0]  # => true
 ```
 """
-struct Stacked{Bs, Rs} <: Transform
+struct Stacked{Bs, Rs<:Union{Tuple,AbstractArray}} <: Transform
     bs::Bs
     ranges::Rs
 end
 Stacked(bs::Tuple) = Stacked(bs, ntuple(i -> i:i, length(bs)))
 Stacked(bs::AbstractArray) = Stacked(bs, [i:i for i in 1:length(bs)])
+Stacked(bs...) = Stacked(bs, ntuple(i -> i:i, length(bs)))
```

Comment:
For a follow-up issue/PR/discussion: I think it would be nice if `ranges` to a collection of integers to avoid these one-element ranges.

Reply:
I'm not entirely certain I understand what you mean :thinking:

Reply:
That it would be nice to just use `ntuple(identity, length(bs))` or maybe even `Base.OneTo(length(bs))` or similar ranges instead of `ntuple(i -> i:i, length(bs))`.

Reply:
Ah yes, I agree. `Stacked` is pretty bad imo, so this is just one out of several improvements we need to make to it.